libafl_frida/src/windows_hooks.rs

```diff
@@ -0,0 +1,62 @@
+// Based on the example of setting hooks: https://github.com/frida/frida-rust/blob/main/examples/gum/hook_open/src/lib.rs
+use frida_gum::{interceptor::Interceptor, Gum, Module, NativePointer};
+use libafl::bolts::os::windows_exceptions::{
+    handle_exception, IsProcessorFeaturePresent, EXCEPTION_POINTERS, PROCESSOR_FEATURE_ID,
+};
+
+/// Initialize the hooks
+pub fn initialize(gum: &Gum) {
+    let is_processor_feature_present =
+        Module::find_export_by_name(Some("kernel32.dll"), "IsProcessorFeaturePresent");
+    let is_processor_feature_present = is_processor_feature_present.unwrap();
+    if is_processor_feature_present.is_null() {
+        panic!("IsProcessorFeaturePresent not found");
+    }
+    let unhandled_exception_filter =
+        Module::find_export_by_name(Some("kernel32.dll"), "UnhandledExceptionFilter");
+    let unhandled_exception_filter = unhandled_exception_filter.unwrap();
```

Comment:
could you return Err(Error::Unknow("Some err msg")) in these locations instead of calling panic!() and unwrap()?

and then you can change the function signature to `pub fn initialize(gum: &Gum) -> Result<(), Error>`

Reply:
Should be fixed now, but that `to_string()` looks a bit ugly to me, is that correct way of calling `Error()`?

Reply:
Could this ever fail? Else a panic would be okay here.
As for the string, the exception owns the string, so that'd be the proper way

Reply:
I think it can't fail unless something is very wrong with the OS. Should I return panic then?

Reply:
Switched to panic back, as discussed with @domenukk, is it ok for you @tokatoka ?

Reply:
sure